src/App.css

```diff
@@ -68,3 +69,27 @@ input {
 .card {
   display: inline-block;
 }
+.result_title{
+   font-size: 32px;
+   font-family: 'Open Sans', sans-serif;
+   font-weight: 700;
+   margin: 0;
+   display: flex;
+   align-items: center;
+   justify-content: center;
+}
```

Comment:
I think the title should have the same color as other lines in result window #444

Reply:
I'm trying to create a distinction for information  hierarchy using font weight and color.